src/Adapter/Shop/QueryHandler/SearchShopsHandler.php

```diff
@@ -59,8 +68,16 @@ public function __construct(ShopRepository $shopRepository)
     public function handle(SearchShops $query): array
     {
         $searchTerm = $query->getSearchTerm();
+        $shopGroupList = $this->shopGroupRepository->findBySearchTerm($searchTerm);
         $shopList = $this->shopRepository->findBySearchTerm($searchTerm);
-        $result = [];
+
```

Comment:
I don't find the behat test relative to this modification ?

Reply:
I did update the behat tests in this file:

`tests/Integration/Behaviour/Features/Scenario/Shop/search_shops.feature`

It now tests results for shops AND shop groups.

Reply:
Sorry, you're right.